Utilities/MCStepLogger/src/analyseCI.cxx

```diff
@@ -0,0 +1,30 @@
+#include <string>
+
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE AnalysisBasicTest
+#include <boost/test/unit_test.hpp>
+#include <boost/test/floating_point_comparison.hpp>
+
+#include "MCStepLogger/AnalysisFile.h"
+
+using namespace o2::mcstepanalysis;
+
+float integral(const TH1* histogram)
+{
+  return histogram->Integral();
+}
+
+BOOST_AUTO_TEST_CASE(analysis_basic_test)
+{
+
+  const std::string analysisName = "AnalysisBasic";
+  BOOST_REQUIRE(boost::unit_test::framework::master_test_suite().argc == 2);
+
+  AnalysisFile file;
+  BOOST_REQUIRE(file.read(boost::unit_test::framework::master_test_suite().argv[1]));
+  BOOST_REQUIRE(analysisName.compare(file.analysisName()) == 0);
+  // count total number of steps including all events
+  TH1* histNSteps = file.getHistogram("nSteps");
+  BOOST_TEST(integral(histNSteps) == 1000000., boost::test_tools::tolerance(0.0005));
```

Comment:
ok. We probably need to make this number configurable or couple the test to a concrete monte carlo production test.